<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editLibrary</class>
 <widget class="QWidget" name="editLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Library</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Selected Template/Library:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentLIbrary">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectTemplateButton">
         <property name="text">
          <string>Select Library/Template</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="returnButton">
       <property name="text">
        <string>Return to Main Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
